cogs/boards.py

```diff
@@ -123,10 +123,15 @@ async def board_loop(self):
                 if "mon_today" in board['type']:
                     mon_today = await queries.statboard_mon_today(self.bot.config, area[0])
                     if "mon_active" in board['type']:
-                        text = f"{text}{self.bot.custom_emotes['pokeball']} **{mon_active[0][0]:,}** {self.bot.locale['active_pokemon']} | **{mon_today[0][0]:,}** {self.bot.locale['today']}\n\n"
+                        scanned_ratio = int(round((mon_today[0][1] / mon_today[0][0] * 100), 0))
+                        text = f"{text}{self.bot.custom_emotes['pokeball']} **{mon_active[0][0]:,}** {self.bot.locale['active_pokemon']} | **{mon_today[0][0]:,}** ({scanned_ratio}% mit IVs) {self.bot.locale['today']}\n\n"
                     else:
                         text = f"{text}{self.bot.custom_emotes['pokeball']} **{mon_today[0][0]:,}** {self.bot.locale['pokemon_seen_today']}\n\n"
-                
+
+                if "hundos_today" in board['type']:
+                    hundos_today = await queries.statboard_hundos_today(self.bot.config, area[0])

```

Comment:
- Hundos will also take ages for larger tables
- You should implement this in the `!board create` command, so you can choose this option when using the command
- Again, this needs better translations (=an extra string in the locale files)
- Maybe also add hundo_active while you're at it?